erpnext/manufacturing/doctype/production_plan/production_plan.py

```diff
@@ -520,3 +487,60 @@ def get_bin_details(row):
 	""".format(conditions=conditions), { "item_code": row.item_code }, as_list=1)
 
 	return item_projected_qty and item_projected_qty[0] or (0,0)
+
+@frappe.whitelist()
+def get_items_for_material_requests(doc, company=None):
```

Comment:
always explicitly check permissions for whitelisted methods.

Reply:
@shreyashah115 Please fix this.